Content.Server/Nutrition/EntitySystems/TrashOnSolutionEmptySystem.cs

```diff
@@ -41,11 +44,11 @@ public void UpdateTags(Entity<TrashOnSolutionEmptyComponent> entity, Solution so
         {
             if (solution.Volume <= 0)
             {
-                _tagSystem.AddTag(entity.Owner, "Trash");
+                _tagSystem.AddTag(entity.Owner, TrashTag);
                 return;
             }
-            if (_tagSystem.HasTag(entity.Owner, "Trash"))
-                _tagSystem.RemoveTag(entity.Owner, "Trash");
+            if (_tagSystem.HasTag(entity.Owner, TrashTag))
+                _tagSystem.RemoveTag(entity.Owner, TrashTag);
```

Comment:
```suggestion
            _tagSystem.RemoveTag(entity.Owner, TrashTag);
```
I think the `HasTag` is not needed here, `RemoveTag` won't throw any errors if it does not exist

Reply:
Yep! Just checked that's correct, I'll add that in too